src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -2378,19 +2476,22 @@ void AutomationInstrumentClipView::modEncoderButtonAction(uint8_t whichModEncode
 			modelStackWithParam->autoParam->deleteAutomation(action, modelStackWithParam);
 
 			display->displayPopup(l10n::get(l10n::String::STRING_FOR_AUTOMATION_DELETED));
```

Comment:
after automation is deleted, or pasted, display and indicators need to be updated